source/_components/light.mipow.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Mipow Color"
+description: "Instructions how to setup Mipow color BT le bulb within Home Assistant."
+date: 2015-10-25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mipow.png
+ha_category: Light
+ha_iot_class: "Local Polling"
+featured: false
+ha_release: xx
+---
+
+The `Mipow color` support is integrated into Home Assistant as a light platform. Please note that this will only work on linux and requires a bluetooth adapter with LE support!
+
+Example of bulbs:
+
+- [Mipow Bulb](https://www.amazon.com/Bluetooth-Controlled-Dimmable-Decorative-Christmas/dp/B014H9YRS0/ref=pd_sbs_107_2?ie=UTF8&psc=1&refRID=8VF91QP5SHY2SJ398RTW)
+
+
+The bulb is controlled normally using Playbulb X app on smartphones
+
+To enable mipow control add following to  `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: mipow
+```
+
+Configuration variables:
+
+- **name** (*Required*): Name for the bulb
```

Comment:
Most platforms has the name variable as an optional variable. I would suggest falling back to device name presented by the Bluetooth protocol or the mac address.